jobs/build/ocp4_scan/Jenkinsfile

```diff
@@ -175,8 +175,19 @@ timeout(activity: true, time: 30, unit: 'MINUTES') {
                                         )
                                         echo stdout
                                     } catch(err) {
-                                        echo "Error checking for INCONSISTENT_RHCOS_RPMS:\n${err}"
-                                        echo "RHCOS builds inconsistent, not triggering build-sync"
+                                        echo "INCONSISTENT_RHCOS_RPMS:\n${err}"
+                                        echo "RHCOS builds inconsistent, not triggering build-sync."
+                                        // inconsistency probably means partial failure and we would like to retry.
+                                        // but don't kick off more if already in progress.
+                                        lock(resource: "rhcos-lock-${version}", skipIfLocked: true) {
+                                            echo "triggering a ${version} RHCOS build for consistency."
+                                            build(
+                                                wait: false,
+                                                propagate: false,
+                                                job: '/aos-cd-builds/build%2Frhcos',
+                                                parameters: [string(name: 'BUILD_VERSION', value: version)]
+                                            )
+                                        }
                                         return
```

Comment:
`currentBuild.description += "<br>triggered rhcos build: ${version}"` would be nice